### DISCUSSION TITLE
Create multiple websites with multiple domains under one GitHub Account

### DISCUSSION THREAD

**Discussion by DavidWarner989** [community/community#30915] [New to GitHub] [answered]
Hello,
I'm brand new to GitHub and am currently floundering my way through the basics. I've managed to host two separate repositories each with their own custom domain under the same GitHub user account. Win. At least I think so... The strange thing about it is that I could find NO OFFICIAL DOCUMENTS from GitHub explaining how to do this. In fact I came across several  articles stating that this wasn't even possible. Can anyone help me clarify if this activity is allowed by GitHub and/or safe and stable in a technical sense? (I know very little about the tech side of hosting). Many thanks David

**Comment by airtower-luna** [comment, marked as answer]
Here's the entry point to the official documentation: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages

In short, you can set one custom domain per repository. If you set one for your user site repository it'll apply to all your Pages sites that *don't* have another custom domain.

**Reply by DavidWarner989** [reply]
> In short, you can set one custom domain per repository. If you set one for your user site repository it'll apply to all your Pages sites that _don't_ have another custom domain.

Thanks this is helpful. Just strange that no where in the official documentation it states this (that I can find at least)

**Reply by airtower-luna** [reply]
It's in there, though not phrased explicitly like that:

* If you enable Pages for any repository you create a Pages site.
* You can set a custom domain for your site.

**Reply by DavidWarner989** [reply]
Ok thanks for your help, that makes more sense :)

**Comment by jfullstackdev** [comment]
you can also create Orgs, they are just like users with fine-grained features for collab, but you can have them even without members,

it also follows the default naming of GitHub Pages ending in `.github.io`, it looks like another user but it's not, it's still you under your account, then, setup all the custom domain you want per Org

take a look at mine even though I don't have a custom domain

https://github.com/devz-throwable

the root of that is 

https://devz-throwable.github.io/

![image](https://user-images.githubusercontent.com/47092464/186937635-87b9d2d2-313d-42c5-a9eb-bb2f3f770c95.png)

it's like another user but it's not, this way I can organize my repos too, so it will not be too many for my personal Repos tab

**Comment by stasyu2009-ux** [comment]
доброго дня всім я програміст на веб розробці та хотів б звами співпрацювати як в команді мова українська.

**Comment by stasyu2009-ux** [comment]
Я шукаю собі онлайн команду.

**Comment by stasyu2009-ux** [comment]
і хочу з кімсь співпрацювати та робити вироби для бізнесу в веб сайт розробці та інших розробок маю досвід знання.

**Comment by stasyu2009-ux** [comment]
Якщо згодні зі мною співпрацювати то пишіть в гід хаб там будем спілкуватися та створювати команду для бізнесу.
маю середню освіту для того щоб співпрацювати я неповнолітній.17 років.

**Comment by stasyu2009-ux** [comment]
Понад сім років досвіду як я програмую
